### DISCUSSION TITLE
可不可以禁用某个输入项，从而让提交更简单？

### DISCUSSION THREAD

**Discussion by erbao2001** [Zhengqbbb/cz-git#95] [Q&A]
比如图中红色框住的选项就都禁用掉，git cz的时候不提示填写了，可以禁用吗？
<img width="635" alt="image" src="https://user-images.githubusercontent.com/123732260/221514687-8e784b30-cac4-4f52-bce5-7bdeab116004.png">

**Comment by Zhengqbbb** [comment]
## `skipQuestions`
https://cz-git.qbb.sh/zh/config/engineer#skipquestions

e.g Code:
https://github.com/Zhengqbbb/qbb.sh/blob/507d1667dbfe97d42d25d28086b86bbd193888df/.czrc#L14

试试看吧 ～

**Reply by itchaox** [reply]
请问下怎么跳过 issue 这个选项呢，我没找到对应的字段呢，谢谢 
（如下这个选项）

 Select the ISSUES type of change (optional): Use arrow keys or type to search
❯ skip 
  custom

**Reply by itchaox** [reply]
@Zhengqbbb

**Reply by Zhengqbbb** [reply]
都试过了吗，有问题可以贴一下配置
```
{
  skipQuestions: ['footerPrefix', 'footer']
}
```

---

<img width="1163" alt="CleanShot 2023-07-20 at 00 39 42@2x" src="https://github.com/Zhengqbbb/cz-git/assets/40693636/2b80ef0b-f970-417d-8f3d-07f00f6d606e">

```
{
  allowCustomIssuePrefix: false,
  allowEmptyIssuePrefix: false
}
```

**Reply by itchaox** [reply]
现在可以了，最开始我没有把 footerPrefix 放入  skipQuestions 中，谢谢哈

**Comment by erbao2001** [comment]
@Zhengqbbb 感谢回答确实可用
